From c956f7522c4ff84ce2f73464b21ee6d33f5f6242 Mon Sep 17 00:00:00 2001
From: Khral Steelforge <garuda2550@gmail.com>
Date: Sat, 23 Feb 2019 18:58:42 +0700
Subject: opencl: Enable platform extensions pass-though.

Signed-off-by: Khral Steelforge <garuda2550@gmail.com>

diff --git a/dlls/opencl/opencl.c b/dlls/opencl/opencl.c
index 2762498468..e5b1fbba82 100644
--- a/dlls/opencl/opencl.c
+++ b/dlls/opencl/opencl.c
@@ -64,6 +64,7 @@ cl_int WINAPI wine_clGetPlatformInfo(cl_platform_id platform, cl_platform_info p
     /* Hide all extensions.
      * TODO: Add individual extension support as needed.
      */
+/*
     if (param_name == CL_PLATFORM_EXTENSIONS)
     {
         ret = CL_INVALID_VALUE;
@@ -82,6 +83,7 @@ cl_int WINAPI wine_clGetPlatformInfo(cl_platform_id platform, cl_platform_info p
         }
     }
     else
+*/
     {
         ret = clGetPlatformInfo(platform, param_name, param_value_size, param_value, param_value_size_ret);
     }
